### PR TITLE
Add docs on how to set up Windows clients

### PR DESCRIPTION
**Pull request by kradalby**

### CODE REVIEW

**Review by juanfont — APPROVED** [review]

File: docs/windows-client.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+in your `headscale` output, turn on `DEBUG` logging and look for:
+
+```
+2022-02-11T00:59:29Z DBG Machine registration has expired. Sending a authurl to register machine=redacted
+```
+
+This typically means that the register keys above was not set appropriatly.
+
+Ensure they are set correctly, delete Tailscale APP_DATA folder and try to connect again.
```
Comment (ryantm) [inline docs/windows-client.md:42]: Should also add "delete the node".

File: docs/windows-client.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+[GIN] 2022/02/10 - 16:39:34 | 200 |    1.105306ms |       127.0.0.1 | POST     "/machine/redacted"
+```
+
+in your `headscale` output, turn on `DEBUG` logging and look for:
+
+```
+2022-02-11T00:59:29Z DBG Machine registration has expired. Sending a authurl to register machine=redacted
+```
+
+This typically means that the register keys above was not set appropriatly.
```
Comment (ryantm) [inline docs/windows-client.md:40]: "register" -> "registry"

File: docs/windows-client.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+## Goal
+
+This documentation has the goal of showing how a user can use the official Windows [Tailscale](https://tailscale.com) client with `headscale`.
+
+## Add registry keys
+
+To make the Windows client behave as expected and to run well with `headscale`, two registry keys **must** be set:
+
+- `HKLM:\SOFTWARE\Tailscale IPN\UnattendedMode` must be set to `always` to allow Tailscale to run properly in the background
+- `HKLM:\SOFTWARE\Tailscale IPN\LoginURL` must be set to `<YOUR HEADSCALE URL>` to ensure Tailscale contacts the correct control server.
```
Comment (ryantm) [inline docs/windows-client.md:12]: say both of these entries should be "string values"

File: docs/windows-client.md (hunk 1 above)
Comment (ryantm) [reply docs/windows-client.md:42]: say 
```
C:\Users\USERNAME\AppData\Local\Tailscale
```

instead of just APP_DATA.

**Pull request merged**